doc/userGuide.md

```diff
@@ -311,6 +311,43 @@ openjdk-tests/test-results/openjdk/report
 
 The JTREG report HTML summary file is then located at `openjdk-tests/test-results/openjdk/report/html/index.html`
 
+
+
+### Use live_monitor feature
+TKG has a script that monitors the test progress live. It can be your only friend in darkest nights. When you run tests locally, you may want to use that feature. It shows how many tests are passed currently etc. Now let's look how to use it.You need Python installed to your machine in order to use it. Note that this feature currently works with openjdk tests only. This means you need to use:	
+
+`export BUILD_LIST=openjdk`
+
+before compiling or you need to run just openjdk tests and pipe them. Otherwise it won't work.
+
+1. You need to change verbose option of jtreg. In order to do that, you need to change 1 line of code on '/openjdk/openjdk.mk' file. Go to the Line 71 and change
```

Comment:
You could likely either hyperlink to the line, or to future proof our documentation, just hyperlink to the file and let the user find the line themselves, as it may change as that file gets edited over the years to be no longer on line 71

Reply:
Done.